docs/guide/api-environment-runtimes.md

```diff
@@ -190,15 +190,10 @@ export interface ModuleRunnerOptions {
     | InterceptorOptions
   /**
    * Disable HMR or configure HMR options.
+   *
+   * @default true
    */
-  hmr?:
-    | false
-    | {
-        /**
-         * Configure HMR logger.
-         */
-        logger?: false | HMRLogger
-      }
```

Comment:
`ModuleRunnerHmr` is not mentioned anywhere on this page. Should we add a type for it?

`serverModuleRunner` also has `logger?:` instead of `ModuleRunnerHmr`